.env-example

```diff
@@ -0,0 +1,18 @@
+# docker
```

Comment:
What does this file exactly does?

Reply:
it provides some variables to the docker-compose. I forgot to mention that you need to rename it to .env in the README.md

for eg, if you wanna setup some auth to mongodb service:
```
# mongodb vars
MONGODB_AUTH=
MONGODB_AUTH_DB=
MONGODB_DATABASE=
MONGODB_DBPATH=
MONGODB_PASSWORD=
MONGODB_REST=
MONGODB_ROOT_PASSWORD=
MONGODB_ROOT_USERNAME=
MONGODB_USERNAME=
MONGODB_VERBOSE=
```